mautic/mautic
# Fix points in webhook after form submission

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | ✅ 
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
If you create form with points change action and try get contact data by webhook, you receive not updated points. This PR fixed it



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create form with points change form action (increase 1 point)
3. Create webhook for form submission - you can use https://webhook.site/ to catch test data
4. Submit the form and check on https://webhook.site/ points are not updated in payload
5. After PR you should receive proper points
